daemon/execdriver/lxc/lxc_template.go

```diff
@@ -160,7 +160,46 @@ lxc.cap.drop = {{.}}
 {{end}}
 `
 
+const LxcInitTemplate = `
+lxc.include = /usr/share/lxc/config/ubuntu.common.conf
```

Comment:
I used the ubuntu conf of lxc to start  /sbin/init  in container with lxc driver; Should i use the content of  conf  instead of include ubuntu conf.